specification/maps/data-plane/Render/preview/2.1/examples/SuccessfulAttributionRequest.json

```diff
@@ -3,7 +3,12 @@
     "api-version": "2.1",
     "tilesetId": "microsoft.base",
     "zoom": 6,
-    "bounds": "-122.414162,47.579490,-122.247157,47.668372"
+    "bounds": [
+      -122.414162,
+      47.579490,
+      -122.247157,
+      47.668372
+    ]
```

Comment:
It is a bit ambiguous how example should write, when the query parameter is of array with CSV (seems both is correct, CSV as string, or array).

If some codegen or MSDocs generator requires this form, I am good with it. But do check the result.

Reply:
MSDocs now requires it, so there is no alternative here. The old way fails model validation step in the pipeline.